ext/microprofile/mp-rest-client/src/main/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
```

Comment:
Remove this file - everything is done through CDI Extension. This may add beans not inteded to be added through discovery